src/main/java/pb360/service/impl/GenerateRestApiServiceImpl.java

```diff
@@ -105,6 +156,21 @@ public MessageObject deleteRestApiData(String restId) {
 		delete.setType("DELETE");
 		delete.setDatetime(getCurrentDateTime().getDatetime());
 
+		@SuppressWarnings("resource")
+		MongoClient mongoClient = new MongoClient("localhost", 27017);
+		MongoDatabase database = mongoClient.getDatabase("restapidbs");
+		MongoCollection<Document> collection = database.getCollection("RestApi");
```

Comment:
I don't know what are you doing Sang

Reply:
This is another way to read document/collection from database.
mongoClient to tell the address
database to get database's name
collection to point to RestApi collection

And from collection i can count/find a field in documents